lib/core_extensions/string.rb

```diff
@@ -0,0 +1,7 @@
+class String
```

Comment:
This seems like a bit much. I don't want monkeypatching in here, we shouldn't need it for any reason. Let's find a different way...

Reply:
I agree, for some reason, it made sense in my mind when I was coding it.